docs/reference-capabilities/consume-and-destructive-read.md

```diff
@@ -23,6 +23,15 @@ fun test(a: Wombat iso) =>
 
 Here's an example of that. When you try to assign `a` to `c`, the compiler will complain.
 
+By default, a `consume` expression returns a type with the capability of the variable that you are assigning to. You can see this in the example above, where we say that `b` is `Wombat iso`, and as such the result of the `consume` expression is `Wombat iso`. We could also have said that `b` is a `Wombat val`, but sometimes, like for example if the type of a variable is quite long, we can instead give an explicit reference capability to the `consume` expression:
```

Comment:
Not sure if that particular motive should be expressed. There could be other reasons to opt for this sugar.

```suggestion
By default, a `consume` expression returns a type with the capability of the variable that you are assigning to. You can see this in the example above, where we say that `b` is `Wombat iso`, and as such the result of the `consume` expression is `Wombat iso`. We could also have said that `b` is a `Wombat val`, but we can instead give an explicit reference capability to the `consume` expression:
```

However, I don't think the example needs to be changed.

Reply:
Sounds good, I've removed it